File: CivitasJ/src/civitas/Prueba.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package civitas;

import civitas.Civitas;
import java.util.ArrayList;
import juegoTexto.Controlador;
import juegoTexto.VistaTextual;

/**
 *
 * @author ELVIRA
 */
public class Prueba {
    private VistaTextual vista;
    private Civitas juego;
    
    Prueba(){
        ArrayList<String> jugadores= new ArrayList<>();
        jugadores.add("Elvira");
        jugadores.add("Ale");
        //jugadores.add("Jolu");
        juego = new Civitas(jugadores);
        vista = new VistaTextual();
    } 

    public VistaTextual getVista() {
        return vista;
    }

    public Civitas getJuego() {
        return juego;
    }
    
    
    
    public static void main(String args[]){
        Dado d= Dado.getInstance();
        d.setDebug(Boolean.TRUE); //comentar para probar el juego completo
        //d.setDebug(Boolean.FALSE);// descomentar para probar el juego completo
        Prueba p1= new Prueba();
       
        Controlador control = new Controlador(p1.getJuego(), p1.getVista());
        control.juega();
    }
}

File: CivitasR/lib/civitas/prueba.rb
# encoding: UTF-8
# 
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.
#require_relative "titulo_propiedad.rb"
#require_relative "mazo_sorpresas.rb"
#require_relative "sorpresa.rb"
#require_relative "tiposorpresa.rb"
require_relative "vista_textual.rb"
require_relative "diario.rb"
require_relative "dado.rb"
require_relative "civitas.rb"
require_relative "controlador"
require_relative "operaciones_juego.rb"
require_relative "jugador.rb"
#require "byebug"

module Civitas
  class Prueba
    
    attr_reader :vista, :juego
    
    def initialize
      jugadores = Array.new
      jugadores << "Elvira"
      jugadores << "Ale"
      #jugadores << "Jolu"
      @vista = Vista_textual.new
      @juego =  Civitas.new(jugadores)
    end
   
    def self.main
      Dado.instance.setDebug(true)
      p1 = Prueba.new
      #byebug
      control =  Controlador.new(p1.juego, p1.vista)
      control.juega
    end
    
  end
  Prueba.main
end
